src/VirtoCommerce.TaxModule.Data/Provider/FixedRateTaxProvider.cs

```diff
@@ -10,9 +10,9 @@ namespace VirtoCommerce.TaxModule.Data.Provider
 {
     public class FixedRateTaxProvider : TaxProvider
     {
-        public FixedRateTaxProvider() : base("FixedRate")
+        public FixedRateTaxProvider(string code)
```

Comment:
Do we need to pass `code` for this provider? Seems it is always should be `"FixedRate"`

Reply:
reverted